Example/Assets/en.lproj/Localizable.strings

```diff
@@ -23,9 +23,8 @@
 "cancel" = "Cancel";
 "yes" = "Yes";
 
-"test_mode_publishing" = "Published";
-"test_mode_testing" = "Testing";
-"test_mode_default" = "Default";
+"test_mode_publishing" = "Hosted apps";
```

Comment:
I think "Published Apps" might be more clear to the user since that is the terminology used elsewhere on the backend.

Reply:
You're right, I was just blindly following tickets instructions 😅